### DISCUSSION TITLE
[BUG]应用市场关于在ghcr.io的镜像拉取问题！

### DISCUSSION THREAD

**Discussion by DracoFeline** [1Panel-dev/1Panel#6519] [Ideas]
### 联系方式

admin@feiyaoblog.com或者admin@feiyaoblog.com.cn

### 1Panel 版本

v1.10.15-lts

### 问题描述

我一开始在应用市场安装umami的时候安装失败，于是我使用了镜像加速，发现没有用。我更换了多台国内的服务器发现均无法拉取镜像。于是我查询了umami的镜像库，发现并不在docker hub上面，而是在ghcr.io上面，国内访问不通畅，速度极慢。于是我直接在镜像加速那里使用ghcr.nju.edu.cn这个加速源，发现拉取镜像依旧走的是ghcr.io，需要手动拉取或者手动去改 docker-compose.yml配置文件
![KMB0W6%9T$@F{HBHAM%AZG6](https://github.com/user-attachments/assets/726e4f17-e332-4624-844c-797745a6b726)
所以希望改进，这也可以算是一个bug。

### 重现步骤

使用国内机器，无论用不用docker hub加速，在应用市场安装，均拉不到ghcr.io上面的镜像。（目前已知umami）

### 期待的正确结果

拉取成功，希望能添加一下加速或者像docker hub那样给一个可以填加速地址的位置，对小白能友好些。

### 附加信息

祝全体1Panel工作人员，开发者。飞致云全体员工中秋快乐，身体健康，长命百岁，升职加薪。祝飞致云公司步步高升，一帆风顺，谢谢！

**Comment by DracoFeline** [comment]
手动拉取镜像指令docker pull ghcr.nju.edu.cn/umami-software/umami:mysql-v2.13.2

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，我们将在后续版本中将 ghcr 仓库的加速地址添加到官方文档中。如果有稳定的加速地址，我们也会考虑在安装 1Panel 系统时默认配置 ghcr 仓库的加速地址。

**Comment by mjsoftking** [comment]
@wanghe-fit2cloud 建议修改为 `umamisoftware/umami` ，配合docker加速地址，下载很快

**Reply by wanghe-fit2cloud** [reply]
这应该不是官方维护的，对吗？
![image](https://github.com/user-attachments/assets/e378fccc-3aff-41cf-9154-d79bd27227da)

> 目前版本，我认为在 `容器` - `配置` 页面添加 ghcr 仓库加速地址可能会更合适一些。

**Reply by mjsoftking** [reply]
@wanghe-fit2cloud 速度够快就行，没看到在哪维护的，最近发现gitea被改成企业版了，没敢更新，不知道社区版用户能否直接免费用

**Reply by wanghe-fit2cloud** [reply]
Gitea: https://github.com/1Panel-dev/1Panel/issues/5516
